app/src/main/java/com/example/parkingsystem/mvp/view/base/ReservationViewBase.java

```diff
@@ -0,0 +1,25 @@
+package com.example.parkingsystem.mvp.view.base;
+
+import android.app.Activity;
+import android.content.Context;
+import androidx.annotation.Nullable;
+import java.lang.ref.WeakReference;
+
+public class ReservationViewBase {
```

Comment:
this class is not necessary, please remove it.
You should use the ActivityView class that you already have

Reply:
Removed